packages/web/components/Rubric.tsx

```diff
@@ -0,0 +1,75 @@
+
+
+interface RubricProps {
+  criteria: Array<{
+      id: string;
+      description: string;
+  }>;
+  scores: Record<string, number>;
+  onScoreChange: (criterionId: string, score: number) => void;
+}
+
+
+const Rubric: React.FC<RubricProps> = ({ criteria, scores, onScoreChange }) => {    // This state will hold the scores for each criterion
+
+  
+  
+  const circleButtonStyle = (score: number, criterionId: string) => ({
+        width: '40px', // Circle diameter
```

Comment:
Btw check out tailwindcss when you get a chance. Not a blocker, but fyi for future:

- https://tailwindcss.com/docs/width
- https://tailwindcss.com/docs/border-radius#rounded-corners
- https://tailwindcss.com/docs/margin